### PR TITLE
add bottom nav for ca/fr/us

### PR DESCRIPTION
**Pull request by nburt**
Language + country selector for whenever we are ready to turn on France / Canada

### CODE REVIEW

**Comment by drwestco** [comment]
A little strange that picking "France" from the country selector leaves me on a page still in English.  Also, need some indicator of current language+country?

File: public/index.html [hunk 1]
```diff
@@ -315,7 +315,33 @@
 </head>
 
 <body>
-
+<nav class="navbar fixed-bottom navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" href="#">#findthemasks</a>
```
Comment (drwestco) [inline public/index.html]: Why is this a link? Placeholder that's eventually going to navigate somewhere?

**Comment by nburt** [comment]
@drwestco yeah sorry, not ready yet I shouldn't opened it. Realized what you commented and working on a js solution to ensure that country + locales stay the same as you navigate

File: public/index.html [hunk 2]
```diff
@@ -315,7 +315,33 @@
...
+
+  <div class="ml-auto">
+    <ul class="navbar-nav">
+      <li class="nav-item dropup">
+        <a class="nav-link dropdown-toggle" href="#" id="locales-dropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Language
```
Comment (drwestco) [inline public/index.html]: "Language"/"Country" labels should be translated.

Comment (drwestco) [reply public/index.html]: As should the popup entries.

**Comment by nburt** [comment]
there is a flash of English when in a foreign language and switching between pages still, that will be fixed with this: https://github.com/findthemasks/findthemasks/pull/302

File: public/locations-list-map.js [hunk 3]
```diff
@@ -35,6 +36,39 @@ function GetCountry() {
   return 'us';
 }
 
+const generateBottomNav = () => {
+  const localeDropdown = document.getElementById('locales-dropdown-selector');
+  const countryDropdown = document.getElementById('countries-dropdown-selector');
+
+  if (localeDropdown && countryDropdown) {
```
Comment (nburt) [inline public/locations-list-map.js]: only render on the index.html page

**Pull request merged**